### DISCUSSION TITLE
Need help syntax LDAP Filter by Group

### DISCUSSION THREAD

**Discussion by antoniusgenta** [snipe/snipe-it#14845] [Q&A]
we tried to synchronize with the active directory server and it was successful, where we saw the synchronization results of all users displayed. What we want to ask is how to synchronize based on certain groups. We tried the following syntax ldap filter but it didn't work.

&(memberOf=CN=Admin,OU=AppGroup,OU=Groups,OU=Accounts,DC=domain,DC=local)

sync success use ldap filter syntax &(cn=*)

![image](https://github.com/snipe/snipe-it/assets/11153332/b087bee1-3210-4507-85b4-3a59e08cae2d)
![image](https://github.com/snipe/snipe-it/assets/11153332/c3242bbf-d7b2-4b33-88c4-d9e74840e1e0)

we use version 6.4.2

Thanks

**Comment by mauro-miatello** [comment]
I asked chatGPT :) and it says to use

`(&(objectClass=user)(memberOf=CN=Admin,OU=AppGroup,OU=Groups,OU=Accounts,DC=domain,DC=local))`

I see also https://confluence.atlassian.com/kb/how-to-write-ldap-search-filters-792496933.html and it seems correct
